Tests/Core/AWSClientRuntimeTests/EventStream/AWSMessageDecoderStreamTests.swift

```diff
@@ -17,7 +17,7 @@ final class AWSMessageDecoderStreamTests: XCTestCase {
         let sut = EventStream.DefaultMessageDecoderStream<TestEvent>(
             stream: bufferedStream,
             messageDecoder: messageDecoder,
-            unmarshalClosure: jsonUnmarshalClosure(responseDecoder: JSONDecoder())
+            unmarshalClosure: TestEvent.unmarshal
```

Comment:
JSON protocols now provide a code-generated unmarshal closure for streaming types.